src/css/config/config.scss

```diff
@@ -125,6 +125,7 @@ $tu-bold-fontWeight:      700 !default;
 $tu-xsmall-fontSize:           scaleFont($fontSize, -2) !default;
 $tu-small-fontSize:            scaleFont($fontSize, -1) !default;
 $tu-base-fontSize:              scaleFont($fontSize, 0) !default;
+$tu-medium-fontSize:            scaleFont($fontSize, 1) !default;
```

Comment:
One thought to skirt the issue - could the footer text work as the `base` size, and let the bold font weight distinguish it?

If not...

This one is tricky...trying to make a numerical size scale arbitrarily align with a sm, md, lg scale. We equated base with medium on our text size utilities. Unfortunately something like `xmedium` or `Xm`, while consistent with small and large, doesn't make a whole lot of sense.

<img width="460" alt="screen shot 2018-05-07 at 2 58 45 pm" src="https://user-images.githubusercontent.com/9888467/39722114-2e76e58c-5207-11e8-8255-c8138616128f.png">

We could keep what you have, and update the text utilities to reflect that change. I do think there is value in having a class labeled as the base size if you want to revert back it.

Sorry, I am thinking this out while writing. If using one of the existing sizes doesn't work I think it's okay to keep what you have for now, and Bill and I probably need to rethink the text scale.

Reply:
I'm totally flexible in how this is done.  If it's easier for now, I can use the base and set it apart with a bold font-weight?  The we can talk about how maybe these font-sizes should operate in the long run?

Reply:
> If it's easier for now, I can use the base and set it apart with a bold font-weight

Yeah, let's just do that for now. Thanks!